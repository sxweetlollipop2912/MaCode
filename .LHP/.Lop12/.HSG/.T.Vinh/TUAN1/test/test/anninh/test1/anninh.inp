5 10
0 10
1 7
8 6
9 8
10 1
2 5
4 8
3 3
7 3
9 9
1 0
2 10
10 10
2 2
6 8
