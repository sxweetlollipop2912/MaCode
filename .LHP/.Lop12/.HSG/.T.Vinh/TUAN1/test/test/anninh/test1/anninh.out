2
3
4
3
1
7
2
2
5
2
